### PR TITLE
Logs, Watch Slices to replace old reducers, Split Tenant Add slice

### PR DESCRIPTION
**Pull request by dvaldivia**
Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Logs/ErrorLogs/ErrorLogs.tsx [hunk 1]
```diff
@@ -169,26 +162,26 @@ const ErrorLogs = ({
...
           setUserAgents(userAgents);
         }
-        logMessageReceived(m);
+        dispatch(logMessageReceived(m));
       };
       c.onclose = () => {
         clearInterval(interval);
         console.log("connection closed by server");
-        setLogsStarted(false);
+        dispatch(dispatch(setLogsStarted(false)));
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Logs/ErrorLogs/ErrorLogs.tsx:170]: Do we need a double dispatch here?

File: restapi/ws_handle.go [hunk 2]
```diff
@@ -134,10 +134,10 @@ func serveWS(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	//// DELETE ME !!!
+	// // DELETE ME !!!
```
Comment (bexsoft) [inline restapi/ws_handle.go:137]:
```suggestion
	// DELETE ME !!!
```

**Review by bexsoft — CHANGES_REQUESTED** [review]
Labels under node selector are not working. Values doesn't change.
Screen Shot 2022-05-25 at 14 41 09

Identity Provider > Built-in, cannot add additional users

Identity Provider > Active Directory, cannot add additional users under List of user DNs

**Review by Alevsk — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**